File: Programming Language(ver.2)/C#/302.cs
using System;

public class TQC302
{
    public static void Main(string[] args)
    {
        int score;
        score = int.Parse(Console.ReadLine());
        Console.Write(compute(score));
    }

    public static int compute(int score) {
        if (score < 0 || score > 100) 
            return -1;
        
        if (score >= 60)
            return score + 5;

        return score + 10;
    }
}

File: Programming Language(ver.2)/C++/302.cpp
#include <iostream>
using namespace std;

int compute(int);
int main()
{
    int n;
    cin >> n;
    cout << compute(n);
    return (0);
}

int compute(int n)
{
    if (n < 0 || n > 100)
        return -1;
    else if (n >= 60)
        return n + 5;
    else
        return n + 10;
}